core/controllers/profile.py

```diff
@@ -311,6 +352,42 @@ def get(self):
 class SignupHandler(base.BaseHandler):
     """Provides data for the editor prerequisites page."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'username': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_valid_username_string'
+                    }]
+                }
+            },
+            'agreed_to_terms': {
+                'schema': {
+                    'type': 'bool'
+                },
+                'default_value': True
+            },
+            'default_dashboard': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': [
+                        constants.DASHBOARD_TYPE_LEARNER,
+                        constants.DASHBOARD_TYPE_CREATOR
+                    ]
+                }
+            },
```

Comment:
I have a confusion . Is `default_value` needed here.

Reply:
Hi @SD-13 I see you have updated the PR, do you still need any help regarding this comment ?

Reply:
@Nik-09 No, I got this.